benchmarks/cugraph/standalone/cugraph_bulk_sampling.py

```diff
@@ -0,0 +1,736 @@
+    )
+
+    parser.add_argument(
+        '--fanouts',
+        type=str,
+        help='Comma separated list of fanouts (i.e. 10_25,5_5_5)',
+        required=False,
+        default='10_25',
+    )
+
+    parser.add_argument(
+        '--batch_sizes',
+        type=str,
+        help='Comma separated list of batch sizes (i.e. 500,1000)',
+        required=False,
+        default='512,1024'
+    )
+
+    parser.add_argument(
+        '--seeds_per_call_opts',
+        type=str,
+        help='Comma separated list of seeds per call (i.e. 1000000,2000000)',
+        required=False,
+        default='524288',
+    )
+    
+    parser.add_argument(
+        '--reverse_edges',
+        action='store_true',
+        help='Whether to reverse the edges for DGL (defaults to False).  Should be True for DGL, False for PyG.',
+        required=False,
+        default=False,
+    )
+
+    parser.add_argument(
+        '--dask_worker_devices',
+        type=str,
+        help='Comma separated list of dask worker devices',
+        required=False,
+        default="0"
+    )
+
+    parser.add_argument(
+        '--random_seed',
+        type=int,
+        help='Random seed',
+        required=False,
+        default=62
+    )
+
+    parser.add_argument(
+        '--persist',
+        action='store_true',
+        help='Will add additional persist() calls to speed up ETL.  Does not affect sampling runtime.',
+        required=False,
+        default=False,
+    )
+
+    parser.add_argument(
+        '--add_edge_types',
+        action='store_true',
+        help='Adds edge types to the edgelist.  Required for PyG if not providing edge ids.',
+        required=False,
+        default=False,
+    )
+
+    return parser.parse_args()
+
+
+# call __main__ function
+if __name__ == "__main__":
+    logging.basicConfig()
+
+    args = get_args()
+    fanouts = [[int(f) for f in fanout.split('_')] for fanout in args.fanouts.split(',')]
+    datasets = args.datasets.split(',')
+    batch_sizes = [int(b) for b in args.batch_sizes.split(',')]
+    seeds_per_call_opts = [int(s) for s in args.seeds_per_call_opts.split(',')]
+    dask_worker_devices = [int(d) for d in args.dask_worker_devices.split(',')]
+
+    client, cluster = start_dask_client(dask_worker_devices=dask_worker_devices, jit_unspill=False, rmm_pool_size=28e9, rmm_async=True)
+    enable_spilling()
+    stats_ls = []
+    client.run(enable_spilling)
+    for dataset in datasets:
+        for fanout in fanouts:
+            for batch_size in batch_sizes:
+                for seeds_per_call in seeds_per_call_opts:
+                    print(f'dataset: {dataset}')
+                    print(f'batch size: {batch_size}')
+                    print(f'fanout: {fanout}')
+                    print(f'seeds_per_call: {seeds_per_call}')
+
+                    if re.match(r'([A-z]|[0-9])+\[[0-9]+\]', dataset):
+                        replication_factor = int(dataset[-2])
+                        dataset = dataset[:-3]
```

Comment:
We seem to trim `replication` factor here for the second iteration which means that the numbers are not correct from the second point onwards

Reply:
fixed

Reply:
This does  not look to be fixed , Can you please check

Reply:
Ahh, Sorry. I see that you now loop on `datasets`

Reply:
Yeah, you were looking at an outdated version.  No worries.